src/unity/python/turicreate/toolkits/one_shot_object_detector/one_shot_object_detector.py

```diff
@@ -0,0 +1,43 @@
+import random as _random
+import turicreate as _tc
+from turicreate import extensions as _extensions
+from turicreate.toolkits._model import CustomModel as _CustomModel
+
+def create(dataset, target, feature=None, batch_size=0, max_iterations=0,
+           seed=None, verbose=True):
+    model = _extensions.one_shot_object_detector()
+    if seed is None: seed = _random.randint(0, (1<<31)-1)
+    # Option arguments to pass in to C++ Object Detector, if we use it:
+    # {'mlmodel_path':'/Users/schhabra/Desktop/apple/turicreate/darknet.mlmodel', 'max_iterations' : 25}
+    augmented_data = model.augment(dataset, target, _tc.SFrame(), seed)
+    model = _tc.object_detector.create(augmented_data)
```

Comment:
I think it's confusing to use the same variable `model` for both the `one_shot_object_detector` proxy and for the wrapped `ObjectDetector` instance

Reply:
Fixed.